src/net/sourceforge/kolmafia/textui/command/FaxbotCommand.java

```diff
@@ -20,7 +21,12 @@ public void run(final String cmd, final String command) {
 
     boolean tried = false;
 
-    for (FaxBot bot : FaxBotDatabase.faxbots) {
+    // Get the faxbot the user last looked at
+    int preferedBot = Math.max(Preferences.getInteger("faxbots"), 0);
+
+    // Iterate from 0
+    for (int i = 0; i < FaxBotDatabase.faxbots.size(); i++) {
+      FaxBot bot = FaxBotDatabase.getFaxbot(i + preferedBot);
```

Comment:
Is there another implementation where the code that iterates from the preferred index only has to be written once?